services/movie_session.py

```diff
@@ -42,3 +42,12 @@ def update_movie_session(
 
 def delete_movie_session_by_id(session_id: int) -> None:
     MovieSession.objects.get(id=session_id).delete()
+
+
+def get_taken_seats(movie_session_id: int) -> list[dict]:
+    tickets = (
+        Ticket.objects
+        .filter(movie_session_id=movie_session_id)
+        .values_list("row", "seat")
+    )
```

Comment:
There is no need to allocate a variable and after that perform a list comprehension, use `values` instead of `values_list` which includes keys by default.